config-connector/solutions/iam/kpt/service-account/README.md

```diff
@@ -0,0 +1,102 @@
+Service Account
+==================================================
+
+# NAME
+
+  service-account
+
+# SYNOPSIS
+
+  Config Connector compatible YAML files to create a service account in your desired project, and grant it a specific role (default to iam.serviceAccountKeyAdmin) on the service account resource level.
```

Comment:
It should be "..., and grant a specific role (defaults to iam.serviceAccountKeyAdmin) to the service account."

It's no longer granting the new service account a role of a given resource, it's granting a given user/group/sa a role of this new service account.

Reply:
Hi based on our off line discussion, I updated this paragraph. But it seems still a little different from your suggestion here. Please take a look and let me know if it makes sense.